packages/arb-avm-cpp/cavm/cmachine.h

```diff
@@ -64,7 +64,8 @@ CMachine* machineClone(CMachine* m);
 CStatus machineCurrentStatus(CMachine* m);
 CBlockReason machineIsBlocked(CMachine* m, int newMessages);
 
-RawAssertion executeAssertion(CMachine* m, const CMachineExecutionConfig* c);
+RawAssertionResult executeAssertion(CMachine* m,
```

Comment:
I don't think it's critical, but I think many of the cgo methods could fail if the db is shutting down so  not sure if we want to do anything for those. The current generic exception handling behavior could be good enough

Reply:
Added special handling in executeAssertion to reduce extraneous error messages when shutting down while under load.